.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+# trailing whitespace can be significant in markdown
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Markdown allows html to be inserted. This shouldn't be an issue. Having whitespace all over the place is definitely worse, and if you are using Github markdown's 2x-trailing-space-newline I can guarantee no-one will understand it easily.